src/packages/core/index.ts

```diff
@@ -2,6 +2,7 @@ import ConnectorLoader from "./src/connector-loader";
 import { ProviderOptions, ServerOptions } from "./src/options";
 import Server from "./src/server";
 
+export { Status } from "./src/server";
```

Comment:
Exporting as I wanted to use the server's actual Status flags in cli instead of hardcoding the status values there.